### ISSUE TITLE
Formalizing game rules/code of conduct

### ISSUE THREAD

**Issue by wei2gao** [hackagotchi/hackagotchi#105]
In light of recent events we really need to formalize what the rules of the game are and what the punishment should be for breaking them. I'm including a few that I think are necessary as a framework for expansion:

Alt account policy: Alternate accounts ("mules") may only be used to store excess items, to circumvent the limitations of Slack. Growing plants or producing items on mules is prohibited.

Spirit of the game: Hackagotchi centers around the interaction between you and your friends. Deliberate attempts to exploit game features to circumvent this intention are discouraged, and gamebreaking exploits including but not limited to alternate account abuse are prohibited. (This can probably be phrased a lot better)

@cedric-h: "it's the implicit responsibility of the player to report anything potentially gamebreaking to the devs, and they acknowledge that the removal of any items in the wake of any such game breaking outcome is to be expected, especially if they failed to report anything to the developers")

**Comment by wei2gao** [comment]
As for punishments, it could go like this:

1st violation: private warning/revocation of violating materials
2nd violation: public callout
3rd violation: ban from the game?

Anything that's egregiously bad can probably warrant an immediate ban

**Comment by Muirrum** [comment]
I'm curious about the "recent events" bit, care to elaborate?

**Comment by LinusS1** [comment]
Also should be noted that you need to follow the Hack Club CoC too. (When hackagotchi gets bigger making our own version of the hackclub CoC might be good also)

@Muirrum check #hackstead/hackstreet: a player had his alt account removed along with the stuff in the alt because alts aren't aloud to have land.

**Comment by wei2gao** [comment]
another one from ced: "you are not entitled to your hackstead. we reserve the right to manipulate any of it at our discretion"

**Comment by Muirrum** [comment]
I agree that this would be a good idea. I'll write up a draft and send it here for y'all to comment on.

**Comment by Muirrum** [comment]
[code-of-conduct.txt](https://github.com/hackagotchi/hackagotchi/files/4908915/code-of-conduct.txt)
I've written this as a first draft. What do y'all think?

cc @cedric-h

**Comment by wei2gao** [comment]
Looks good so far. I would add a guideline for how punishments/"intervention" will be handled, just to keep admins accountable.

**Comment by Muirrum** [comment]
right, i'll get on that

What do you think accountability should look like?

**Comment by Muirrum** [comment]
Added an "Intervention" section here: https://files.devosmium.xyz/ae0k7d6u.md

**Comment by wei2gao** [comment]
> right, i'll get on that
> 
> What do you think accountability should look like?

Accountability is basically some way to prevent things that could be seen as admin abuse. In particular it might be nice to have a way to appeal interventions if the user believes it was done in bad faith.

**Comment by Muirrum** [comment]
I know what accountability is aha.

I agree that an appeals method would be nice, I was asking your thoughts on what that would look like.

**Comment by wei2gao** [comment]
It wouldn't be mentioned in the rules, but if an admin were to take someone's items or ban a player, they'd need to give written notice of what happened (privately) and give an option to reach out for appeals. (Repeat offenders would probably not get this option)

Also, it should be part of the standard protocol that you have to give written warning to someone before you take their stuff, not the other way around.

**Comment by LinusS1** [comment]
Another thing to add is that repeatedly DMing anyone (players/admin) for things is not allowed, and for repeat offenders there might be consequences (things deleted from hackstead)

**Comment by Muirrum** [comment]
[code-of-conduct.pdf](https://github.com/hackagotchi/hackagotchi/files/4979584/code-of-conduct.pdf)
Here's the updated version.

**Comment by lukec11** [comment]
Current status on this Code of Conduct? Content is fine by me (though I think it should be uploaded in a format easier to PR to, like markdown - happy to convert it over).

**Comment by Muirrum** [comment]
[code-of-conduct.pdf](https://github.com/hackagotchi/hackagotchi/files/5003346/code-of-conduct.pdf)
This is the latest version.

I've got the raw LaTeX file too that I was planning on checking in to a repository but I couldn't figure out where it should go.

**Comment by LinusS1** [comment]
I think that it should be converted to markdown - we use markdown everywhere else, and since clients will need to include this, it would be nice just to need a markdown formatter, not also a latex one also.
